chapters/arrays.tex

```diff
@@ -195,10 +195,14 @@ \section{Built-in Array Functions}\doublelabel{built-in-array-functions}
 \lstinline[mathescape=true]!size(C, $j$)! = $1$ for $n_{\mathrm{A}} + 1 \leq j \leq n$, \lstinline[mathescape=true]!C[$i_{1}$, $\ldots$, $i_{n_{\mathrm{A}}}$, 1, $\ldots$, 1]! =
 \lstinline[mathescape=true]!A[$i_{1}$, $\ldots$, $i_{n_{\mathrm{A}}}$]!\\ \hline
 \end{longtable}
+The argument $n$ must be literal or a constant that can be evaluated during translation because it determines
```

Comment:
For some reason I want to use since instead of because, but I know because is not wrong...

Reply:
I'm with you here, @sjoelund, but i'd go as far as to suggest _as_ with a comma:
```suggestion
The argument $n$ must be literal or a constant that can be evaluated during translation, as it determines
```